src/main/kotlin/ch/hevs/cloudio/cloud/apiutils/EndpointManagementUtil.kt

```diff
@@ -75,16 +83,61 @@ object  EndpointManagementUtil{
 
         val attribute = getAttribute(endpointEntityRepository, AttributeRequest(attributeSetRequest.attributeTopic))
 
+        //only set attribute if setpoint or parameter
         if (attribute == null)
             return ApiActionAnswer(false, "Attribute doesn't exist")
         else if(attribute.constraint != AttributeConstraint.SetPoint && attribute.constraint != AttributeConstraint.Parameter)
             return ApiActionAnswer(false, "Attribute is nor a SetPoint, neither a Parameter")
         else {
+            //send message to amq.topic queue
             rabbitTemplate.convertSendAndReceive("amq.topic",
```

Comment:
If you do not expect an answer, use the convertAndSend() method.

Reply:
Fixed in 0b262d6